src/unicodeMake.js

```diff
@@ -0,0 +1,59 @@
+/* eslint no-console:0 no-undef:0 */
+
+// This is an internal Node tool, not part of the KaTeX distribution,
+// whose purpose is to generate unicodeSymbols.js in this directory.
+// In this way, only this tool, and not the distribution/browser,
+// needs String's normalize function.
+
+const fs = require('fs');
+
+// Read supported accents from Parser.js
+const Parser = fs.readFileSync('./Parser.js', {encoding: 'utf8'});
+const match = /accents = [^;]*;/.exec(Parser);
+eval(match[0]);
```

Comment:
Maybe we could have `Parser.js` export `accents` instead.

Reply:
I should add a comment about this. I originally tried it that way, but Node doesn't support `import`, and I'm running `unicodeMake.js` in Node, so Babel would have to get involved, and the accents are not available in built `katex.js`.  I'm open to better suggestions!  We could, for example, put `accents` here in `unicodeMake.js` and/or `unicodeSymbols.js` and import them in `Parser.js` -- perhaps that's best.

Reply:
OK, I tried moving them. It's not especially pretty -- I still `eval`d the code in `unicodeMake`, which led to extra `\` escaping in the `accents` definition... Let me know if you see a cleaner way! But at least I'm no longer parsing `Parser.js` with a regex!